packages/react-log-viewer/src/LogViewer/examples/LogViewer.md

```diff
@@ -16,8 +17,25 @@ import { data } from './realTestData.js';
 import React from 'react';
 import { data } from './data/realTestData.js';
 import { LogViewer } from '@patternfly/react-log-viewer';
+import { Checkbox } from '@patternfly/react-core';
 
-<LogViewer hasLineNumbers={false} height={300} data={data.data}/>;
+BasicLogViewer = () => {
+  const [isDarkTheme, setIsDarkTheme] = React.useState(false);
+  
+  return (
+    <>
+      <Checkbox
+        label="Dark theme"
+        isChecked={isDarkTheme}
+        onChange={setIsDarkTheme}
+        aria-label="toggle dark theme checkbox"
+        id="toggle-dark-theme"
+        name="toggle-dark-theme"
+      />
+      <LogViewer hasLineNumbers={false} height={300} data={data.data} isDarkTheme={isDarkTheme} />;
```

Comment:
```suggestion
      <LogViewer hasLineNumbers={false} height={300} data={data.data} isDarkTheme={isDarkTheme} />
```
It has a different meaning to the previous usage? Cause I can't see it in the main branch right now, but preview for this PR shows the semicolon.